psi4/driver/qcdb/dashparam.py

```diff
@@ -210,6 +210,21 @@
         'pbe0'        : {'s6': 1.000, 's8': 0.528823, 'a1': 0.007912, 'a2': 6.162326},
         'lcwpbe'      : {'s6': 1.000, 's8': 0.906564, 'a1': 0.563761, 'a2': 3.593680},
     },
+    'nl': { # mostly taken from collected list at http://pubs.acs.org/doi/full/10.1021/acs.jctc.5b01066
+        'blyp'        : {'b': 4.000, 'c': 0.0093},
+        'revpbe'      : {'b': 3.700, 'c': 0.0093},
+        'b3lyp'       : {'b': 4.800, 'c': 0.0093},
+        'b3pw91'      : {'b': 4.500, 'c': 0.0093},
+        'revpbe0'     : {'b': 4.300, 'c': 0.0093},
+        'bp86'        : {'b': 4.400, 'c': 0.0093},
+        'pbe0'        : {'b': 6.900, 'c': 0.0093},
+        'pbe'         : {'b': 6.400, 'c': 0.0093},
+        'tpss0'       : {'b': 5.500, 'c': 0.0093},
+        'dsd-pbepbe'  : {'b': 9.600, 'c': 0.0093},
+        'dsd-pbeb95'  : {'b': 12.500, 'c': 0.0093},
```

Comment:
These three will have to be defined in `dict_dh_funcs` manually, unless they happen to use the dispersionless variant anyway!